js/components/sortable-densities.js

```diff
@@ -35,7 +35,7 @@ export default class SortableDensites extends Component {
 
     // Aftering a reload, plate IDs continue to increment
     // Subtracting the smallest ID ensures visible plate numbering starts at 1
-    let minId = Object.keys(plateDensities).sort()[0] - 1
+    let minId = Object.keys(plateDensities).reduce(function (a, b){ return parseInt(a) < parseInt(b) ? a : b }) - 1
```

Comment:
@ekosmin, `sort` also accepts comparison function, but that is fine too (fancy). ;) 
You could even make it shorter using es6 syntax:
`reduce((a, b) => parseInt(a) < parseInt(b) ? a : b)`
😉

Reply:
Thanks for running the check on the different import styles, I'll keep this in mind! And true, would have been a good use of es6